airbyte-integrations/connectors/source-file/source_file/source.py

```diff
@@ -96,12 +96,23 @@ def check(self, logger, config: Mapping) -> AirbyteConnectionStatus:
         """
         config = self._validate_and_transform(config)
         client = self._get_client(config)
-        logger.info(f"Checking access to {client.reader.full_url}...")
+        source_url = client.reader.full_url
```

Comment:
These changes should be covered by unit test, could you please make a quick parametrized test for this?